decidim-dipgra_census_authorization/app/services/dipgra_census_authorization_handler.rb

```diff
@@ -54,11 +54,13 @@ def census_for_user
     return @census_for_user if defined? @census_for_user
     return unless organization
 
-    @service = DipgraCensusAuthorization.new(DipgraCensusAuthorizationConfig.api_config(organization))
+    username = Rails.application.secrets.dipgra_census[:username]
+    password = Rails.application.secrets.dipgra_census[:password]
+    @service = DipgraCensusAuthorization.new(username: "#{organization.ine_code}#{username}", password:, organization:)
```

Comment:
why this change?

Reply:
Because after update from Ruby 2.7 to Ruby 3.1 old syntax stopped working, so I had to rewrite some partes of this module

Reply:
what was the mistake? it's weird that it doesn't work